client/app/services/projectMap.service.js

```diff
@@ -0,0 +1,142 @@
+(function () {
+    'use strict';
+    /**
+     * @fileoverview This file provides a project map service. It visualises the projects on the map
+     */
+
+    angular
+        .module('taskingManager')
+        .service('projectMapService', [projectMapService]);
+
+    function projectMapService() {
+        
+        var map = null;
+        var projectVectorSource = null;
+        var projectHighlightVectorSource = null;
+        
+        var service = {
+            initialise: initialise,
+            showProjectsOnMap: showProjectsOnMap,
+            highlightProjectOnMap: highlightProjectOnMap,
+            removeHighlightOnMap: removeHighlightOnMap
+        };
+
+        return service;
+
+        /**
+         * Initialises the service by adding the vector layers
+         * @param mapForProjects - OL map 
+         */
+        function initialise(mapForProjects){
+            map = mapForProjects;
+            addProjectsVectorLayer_();
+            addHighlightsVectorLayer_();
+        }
+        
+        /**
+         * Add vector layer for the project results
+         * @private
+         */
+        function addProjectsVectorLayer_(){
+
+            var fill = new ol.style.Fill({
```

Comment:
Done. Also renamed a few functions to make the style service a bit easier to read/understand.